resources/Common/Platform/Windows/default.mk

```diff
@@ -15,8 +16,8 @@ _METADATA_FILE := $(shell \
 	} \
 )
 
-# Override the shell-based metadata functions with PowerShell equivalents
-# These now use the cached metadata file path
+# For backwards compatibility, these functions still work but are no longer used by print_help_header
+# They are kept in case other parts of the codebase call them
```

Comment:
Don't keep anything which is not directly used, no other parts of the codebase calls them, they're specifically used here.